contrib/build-push-ceph-container-imgs.sh

```diff
@@ -378,6 +378,7 @@ function create_registry_manifest {
         DOCKER_IMAGES="$DOCKER_IMAGES ${TARGET_RELEASE}-aarch64"
       fi
 
+      rm -rvf ~/.docker/manifests/*"$(echo "$TARGET_RELEASE" | cut -d ":" -f2)"*
```

Comment:
I think it's safe to remove the entire directory

```suggestion
      rm -rvf ~/.docker/manifests
```